docs/monthly-meeting/2025-04.md

```diff
@@ -0,0 +1,178 @@
+
+### Ensemble contributor session
+
+- Blaise noted he will host a 90-min ensemble session for intermediate to advanced contributors this Friday,
+  as an exercise to get folk used to using Python codebase.
+- The topic will be related to https://github.com/python/cpython/issues/127833 but there is no rigid agenda.
+- Invitation: https://calendar.app.google/5hjYyVRSjJN5MwdEA
+
+### Editorial Board update & documentation audits
+
+- Ned gave a brief update from the Editorial Board.
+- Joanna Jablonski (editor, *Real Python*) and Savannah Ostrowski (core developer) are conducting *documentation audits*.
+- These are a formal approach to assessing gaps in documentation and approaching from disciplined first principles.
+- The pair are planning an audit of the *argparse* docs, with the indent to repeat the process elsewhere in the documentation.
+
+### Potential PR process improvements
+
+- What process changes could we explore to increase velocity of merged docs PRs?
+  Ideally 'easy wins' rather than sweeping changes.
+- Should we suggest:
+
+  * making a single large PR containing all proposed changes?
+  * making many small PRs with individual changes?
+  * simply waiting for a reviewer?
+
+- The open documentation PRs are slowly increasing, and now at around 240.
+- Ned notes that a frequent blocker is that a seemingly small change can represent a philosophical decision, which take time to reach consensus on.
+  Experienced reviewers can spot these types of change quite quickly. How do we communicate this to new contributors?
+- It is disheartening to have a first PR languish unreviewed, or have 70 comments of back-and-forth debate.
+- Hugo noted the exemplar of the *string examples* PR, seeking to add an example snippet to each `str` method.
+  This is unreviewable with everything in one PR, but e.g. splitting to many small PRs might help refine the process.
+  Just a single change request blocks an entire PR, whereas smaller PRs are easier to merge quickly.
+
+- Perhaps we could experiment with defining a tracker issue and splitting it into small tasks.
+  Coordination problems or design decisions can be solved on the tracker issue.
+- Daniele noted that if this got traction, many small atomic changes could be a good approach.
+  Each might look banal, but are part of a larger whole, and once the first few are approved,
+  the reviewer starts to see patterns, reviewing is very quick, and the review experience is nicer.
+- Many small PRs might look like more overhead, but it's not.
+- Perhaps we could try this with the `str` examples PR?
+
+### Synchronous contributions
+
+- Blaise noted that the experience of the Pycon sprints was valuable, being able to do work, get stuck, and then carry on.
+  These settings give the freedom to interrupt and get help from more experienced contributors.
+- Technical writers, such as those in the *Write the Docs* slack, would like to help, but don't know where to begin, and a session like this could help.
```

Comment:
```suggestion
- Technical writers, such as those in the [*Write the Docs* Slack](https://www.writethedocs.org/slack/), would like to help, but don't know where to begin, and a session like this could help.
```